scoresheets/CleanTable.tex

```diff
@@ -2,18 +2,17 @@
 
 \begin{scorelist}
 	\scoreheading{Main Goal}
-	\scoreitem[5]{60}{Picking tableware and cutlery for transportation to the dishwasher}
```

Comment:
To encourage picking up the objects I think we should keep the partial scoring, we may spread the focus more evenly between pickup and place. Removing the partial scoring only discourages going for pickup as you wont get points anymore if the placing fails afterwards.

Reply:
Pick-and-place is now a basic skill (or should be), is more a problem than a solution, i.e. to decide when the limits of a "pick" and a "place" are is ambiguous and hard to evaluate objectively.